src/twisted/web/test/test_http_headers.py

```diff
@@ -371,6 +371,9 @@ def test_getRawHeadersDefaultValue(self):
         h = Headers()
         default = object()
         self.assertIdentical(h.getRawHeaders(u"test", default), default)
+        self.assertIdentical(h.getRawHeaders(u"test", None), None)
+        self.assertEqual(h.getRawHeaders(u"test", [None]), [None])
+        self.assertEqual(h.getRawHeaders(u"test", [u"\N{SNOWMAN}"]), [u"\N{SNOWMAN}"])
```

Comment:
This is fine, but I think that these new last test should be done in a separate test function, with a docstring describing the reason why we have the tests in the first place.

I would argue that the docstring for `getRawHeaders` should be updated from 

```
@rtype: L{list} of strings, same type as C{name}
```

to

```
@rtype: L{list} of strings, same type as C{name} with the exception of the default value, which is returned as such.
```